optima/parameters.py

```diff
@@ -126,14 +126,43 @@ def fixprops(self, fix=None, which=None, startyear=None):
         if   which is None:  which = ['tx','supp']
         elif which is 'all': which = ['dx','tx','supp']
         else:                which = promotetolist(which)
-        if fix:  startyear = self.pars['numtx'].t['tot'][-1]
-        else:    startyear = 2100
+        if startyear is None:
+            if fix:  startyear = self.pars['numtx'].t['tot'][-1]
+            else:    startyear = 2100
         if 'dx'   in which: self.pars['fixpropdx'].t   = startyear
         if 'tx'   in which: self.pars['fixproptx'].t   = startyear
         if 'supp' in which: self.pars['fixpropsupp'].t = startyear # Doesn't make sense to assume proportion on treatment without assuming proportion suppressed....also, crashes otherwise :)
         return None
         
     
+    def usedataprops(self, use=None, which=None):
```

Comment:
@robynstuart i don't know about this...i think we should instead have this in the `Testing & treatment` sheet maybe

Reply:
Let's follow up over email

Reply:
@cliffckerr I believe we resolved this and are going to keep this - ok?